.gitmodules

```diff
@@ -75,3 +75,9 @@
 [submodule "hidapi-src"]
 	path = Externals/hidapi/hidapi-src
 	url = https://github.com/libusb/hidapi
+[submodule "Externals/openxr/OpenXR-SDK"]
+	path = Externals/openxr/OpenXR-SDK
+	url = git@github.com:KhronosGroup/OpenXR-SDK.git
+[submodule "Externals/openxr/OpenXR-SDK-Source"]
+	path = Externals/openxr/OpenXR-SDK-Source
+	url = git@github.com:amwatson/OpenXR-SDK-Source.git
```

Comment:
I think the issue is that you used an incorrect URL, you should use the HTTPS one.

Reply:
Thanks, I fixed this.